config/spec.json

```diff
@@ -39,6 +39,13 @@
     "eip145Transition": "0x38ada7",
     "eip1014Transition": "0x38ada7",
     "eip1052Transition": "0x38ada7",
+    "eip1283Transition": "0xd59f80",
+    "eip1344Transition": "0xd59f80",
+    "eip1706Transition": "0xd59f80",
+    "eip1884Transition": "0xd59f80",
+    "eip2028Transition": "0xd59f80",
+    "eip2929Transition": "0xd59f80",
+    "eip2930Transition": "0xd59f80",
     "maxCodeSize": 24576,
```

Comment:
This is still relevant for OE?
From what you told me that is configured on the node client now

Reply:
I made a mistake here, maxCodeSizeTransition is used to signal the block at which you want to update code size to the value set by maxCodeSize (so will be a fork). Want to bump this up for the fork at 14million or further down the line?